src/goto-symex/symex_goto.cpp

```diff
@@ -32,12 +32,10 @@ void goto_symext::symex_goto(statet &state)
   state.rename(new_guard, ns);
   do_simplify(new_guard);
 
-  if(new_guard.is_false() ||
-     state.guard.is_false())
+  if(new_guard.is_false())
```

Comment:
? Unintentional behaviour change? This now happens for `new == false, state == false`, whereas before it only happened for `new == false, state != false`

Reply:
oh yes it should be `new_guard.is_false && !state.guard.is_false`